js/graviex.js

```diff
@@ -0,0 +1,1012 @@
+        if (cancelRequested) {
+            status = 'canceled';
+        } else if (status === undefined) {
+            status = this.parseTransactionStatus (statusCode);
+        }
+        return {
+            'info': transaction,
+            'id': id,
+            'txid': txid,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'confirmations': confirmations,
+            'type': type,
+            'amount': amount,
+            'currency': code,
+            'status': status,
+        };
+    }
+
+    parseTransactionStatus (status) {
+        const statuses = {
+            'initiated': 'pending',
+            'needs_create': 'pending',
+            'credited': 'ok',
+            'confirmed': 'ok',
+        };
+        return this.safeString (statuses, status, status);
+    }
+
+    async fetchDeposit (id, code = undefined, params = {}) {
+        if (id === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchDeposit() requires a txid argument');
+        }
+        await this.loadMarkets ();
+        const request = {
+            'txid': id,
+        };
+        const response = await this.privateGetDeposit (this.extend (request, params));
+        // {
+        //   "id": 4831985,
+        //   "currency": "ltc",
+        //   "amount": "0.14108256",
+        //   "fee": "0.0",
+        //   "txid": "c0d50a556a48e16f86b571651fd6199ce122ace9e2843e71e953b22db3eaf007",
+        //   "uid": null,
+        //   "created_at": "2022-02-01T18:10:32+03:00",
+        //   "confirmations": "6",
+        //   "done_at": null,
+        //   "state": "accepted"
+        // }
+        // const data = this.safeValue (response, 'data', {});
+        // const deposit = this.safeValue (data, 'deposit', {});
+        return this.parseTransaction (response);
+    }
+
+    async fetchDeposits (code = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let currency = undefined;
+        if (code !== undefined) {
+            currency = this.currency (code);
+        }
+        const response = await this.privateGetDeposits (params);
+        // const data = this.safeValue (response, 'data', {});
+        // const deposits = this.safeValue (response);
+        // [
+        //     {
+        //         "id": 4831985,
+        //         "currency": "ltc",
+        //         "amount": "0.14108256",
+        //         "fee": "0.0",
+        //         "txid": "c0d50a556a48e16f86b571651fd6199ce122ace9e2843e71e953b22db3eaf007",
+        //         "uid": null,
+        //         "created_at": "2022-02-01T18:10:32+03:00",
+        //         "confirmations": "6",
+        //         "done_at": null,
+        //         "state": "accepted"
+        //     }
+        // ]
+        return this.parseTransactions (response, currency, since, limit);
+    }
+
+    parseOrder (order, market = undefined) {
+        // [
```

Comment:
remove these [] pls since the parser receives a single object not a list

Reply:
Okay. Fixed in the latest commit https://github.com/ccxt/ccxt/commit/bb2b0b2ef29c557eee9dad97fee9c010de932c9a